source/ed/connectors/fusio_parser.cpp

```diff
@@ -1543,29 +1543,9 @@ void GridCalendarTripExceptionDatesFusioHandler::handle_line(Data&, const csv_ro
 }
 }  // namespace grid_calendar
 
-void AdminStopAreaFusioHandler::init(Data& data) {
+void AdminStopAreaFusioHandler::init(Data&) {
```

Comment:
You may remove the `Data&` from the signature then, no? Or is it kinda templated and you can't?

Reply:
yes, it's a Template story... 
I can't suppress **Data** in the params function because we use `init()` method in a template context, even if this is not use. 
Just remove the var name to avoid gcc warning. https://github.com/CanalTP/navitia/blob/dev/source/ed/connectors/gtfs_parser.h#L430